gradle/libs.versions.toml

```diff
@@ -13,8 +13,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 [versions]
-adobe-testing-s3mock = "2.17.0"
```

Comment:
Does this PR need to be merged with main? I would expect much fewer changes in dependencies outside of Jetty. It looks like there are other lingering changes that might also need to be up to date with main.